enhancements/plain-k8s-bundles.md

```diff
@@ -0,0 +1,220 @@
+---
+title: plain-k8s-bundles
+authors:
+  - "@exdx"
+  - "@njhale"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2020-03-09
+last-updated: 2020-03-09
+status: implementable
+see-also:
+  - "https://docs.google.com/document/d/1kDDVVdjSjBAP3-4QnIPb67QfcR2-LEWRbvKFUDzEwrc/edit#heading=h.70izt81ko44h"
```

Comment:
Is this doc publicly available?

Reply:
It is not available publicly.

Reply:
I can either drop this doc entirely or make a public copy of it. Do we really need it if this enhancement explains everything?

Reply:
I think its ok to remove the reference to the doc. We have it around internally if we need to review it, but from what I've seen its all been brought in via this enhancement

Reply:
+1 lets just remove the link